backend/Dockerfile

```diff
@@ -9,7 +9,7 @@ ENV PYTHONUNBUFFERED 1
 
 
 RUN pip install --upgrade pip
-COPY requirements.txt requirements.txt
+COPY py-cinema-full-stack/requirements.txt requirements.txt
```

Comment:
failed to build up the container 
the command is attempting to copy a requirements.txt file from a folder called py-cinema-full-stack into the working directory of the container. However, based on your directory structure, requirements.txt is in the root directory of your project